fitbenchmarking/fitbenchmarking
# Moving table generation into results_processing

#### Description of Work

Fixes #411 

Moves table generation out of `core` and into `results_processing`

#### Testing Instructions

1. Check tables are consistent with master

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
